packages/flutter_tools/test/integration.shard/android_plugin_skip_unsupported_test.dart

```diff
@@ -147,12 +147,12 @@ void main() {
     final ProcessResult buildApkResult = await testUnsupportedPlugin(
         project: project, createAndroidPluginFolder: true);
     expect(buildApkResult.stderr.toString(),
-        isNot(contains('Please fix your settings.gradle.')));
+        isNot(contains('Please fix your settings.gradle')));
     expect(buildApkResult, const ProcessResultMatcher());
   });
 
   // TODO(54566): Remove test when issue is resolved.
-  /// Test project with a `settings.gradle` (PluginEach) that apps were created
+  /// Test project with a `settings.gradle` (PluginEach) that apps were cresated
```

Comment:
Think this is a typo. Also thank you for working on this.

Reply:
thanks!